### PR TITLE
[OLs v0.1.1 ] added upgrade graph with skipRange

### PR DESCRIPTION
**Pull request by xrajesh**
## Description

Added skipRange for upgrade 
Picked the service SHA which we released.

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library

## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

File: lightspeed-catalog-4.16/index.yaml [hunk 1]
```diff
@@ -62,3 +62,4 @@ package: lightspeed-operator
 name: preview
 entries:
   - name: lightspeed-operator.v0.1.1
+    skipRange: ">=0.1.0 <0.1.1"
```
Comment (joshuawilson) [inline lightspeed-catalog-4.16/index.yaml:65]: Does this mean, skip "0.1.0"?
Why not just use "=0.1.0"?

Comment (xrajesh) [reply lightspeed-catalog-4.16/index.yaml:65]: From the doc 
```
The entry specifying the skipRange will be presented as a direct (one hop) update to any version from that package within that range. The versions in this range do not need to be in the catalog in order for bundle addition to be successful. We recommend avoiding using unbound ranges such as <1.0.3.
```

**Pull request merged**